src/Skia/Avalonia.Skia/FormattedTextImpl.cs

```diff
@@ -30,6 +30,11 @@ public class FormattedTextImpl : IFormattedTextImpl
 
             SKTypeface skiaTypeface = TypefaceCache.Default;
 
+            if (typeface == null)
```

Comment:
I think it is an error if FormattedTextImpl is created with a null TypeFace. If we keep this you should add this to Direct2D1 too.

Reply:
@Gillibald You are correct, **FormattedTextImpl** shouldn't be created with a null TypeFace.

I will remove this code and will add correct *ScreenPage.cs* as a cleaner workaround.